### ISSUE TITLE
Create app structure

### ISSUE THREAD

**Issue by rajpranz** [chingu-voyage4/Geckos-Team-20#9]
Create basic structure for the application. Setup folder structure to help get the application started

**Comment by TheCandiko** [comment]
Hi there, how is it going with project folder structure? Do you have any progress?

I've got something in my mind. I'm planning with Node.js server, setup for Git, ESLinter and Prettier as well. General folder structure can be like these:

```
glorious-pins
  ├───node_modules
  ├───public
      ├───css
      └───js
  └───src
      ├───config
      ├───FeatureFolders
      ├───model
      └───router
  ```

**Folder descriptions:**
node_modules: Dependency files
public: Client-side files
views: HTML files
css: CSS Style files
js: Client-side JS files
src: Server-side files
config: Configuration files for server
model: Model files for databases
FeatureFolders: Folders for features like `users/`, `pins/` or `/boards`. Probably those folders contain HTML file to render or send, and business logic files. But client-side JS files will will be under `public/js/...`
router: Route files

If you have progressed please share with me, if you don't let me know, I can make it today and push it to repo :)

**Comment by TheCandiko** [comment]
Close issue, resolved by PR #10
